### PR TITLE
Update locality lb docs

### PR DESCRIPTION
**Pull request by howardjohn**
We didnt document all requirements, distribute feature, and we added Helm support

### CODE REVIEW

File: content/help/ops/traffic-management/locality-load-balancing/index.md [hunk 1]
```diff
@@ -50,11 +54,42 @@ The hierarchy of prioritization matches in the following order:
...
 would not improve service health or many other reasons including regulatory controls.
-To constrain traffic to a region, use the mesh `LocalityLoadBalancerSetting.Failover`
-configuration detailed in the [Locality load balancing reference guide](/docs/reference/config/istio.mesh.v1alpha1/#LocalityLoadBalancerSetting).
+To constrain traffic to a region, configure the `localityLbSetting` option using Helm. See the
+[Locality load balancing reference guide](/docs/reference/config/istio.mesh.v1alpha1/#LocalityLoadBalancerSetting)
+for options.
+
+An example configuration:
+
+```yaml
```
Comment (geeknoid) [inline content/help/ops/traffic-management/locality-load-balancing/index.md]: Please use {{< text yaml >}} instead, here and elsewhere

File: content/help/ops/traffic-management/locality-load-balancing/index.md [hunk 2]
```diff
@@ -50,11 +54,42 @@ The hierarchy of prioritization matches in the following order:
...
+global:
+  localityLbSetting:
+    failover:
+    - from: us-east
+      to: eu-west
+    - from: us-west
+      to: us-east
+```
+
+## Locality-Weighted Load Balancing
```
Comment (geeknoid) [inline content/help/ops/traffic-management/locality-load-balancing/index.md]: Only capitalize first character in headings:

Locality-weighted load balancing

File: content/help/ops/traffic-management/locality-load-balancing/index.md [hunk 3]
```diff
@@ -50,11 +54,42 @@ The hierarchy of prioritization matches in the following order:
...
+
+Locality-Weighted Load Balancing distributes user defined percentages of traffic to certain localities.
```
Comment (geeknoid) [inline content/help/ops/traffic-management/locality-load-balancing/index.md]: Locality-weighted load balancing
user-defined

File: content/help/ops/traffic-management/locality-load-balancing/index.md [hunk 4]
```diff
@@ -20,16 +20,20 @@ the geographic location where requests are sent.
...
 Currently, the service discovery platform populates the locality automatically.
 In Kubernetes, a pod's locality is determined via the [well-known labels for region and zone](https://kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#failure-domain-beta-kubernetes-io-region)
 on the node it is deployed. If you are using a hosted Kubernetes service your cloud provider
 should configure this for you. If you are running your own Kubernetes cluster you will need
 to add these labels to your nodes. The sub-zone concept doesn't exist in Kubernetes.
 As a result, this field does not need to be configured.
 
+In order for Istio to determine locality, a Service must be associated with the caller.
+
 ## Locality-Prioritized Load Balancing
```
Comment (geeknoid) [inline content/help/ops/traffic-management/locality-load-balancing/index.md]: Should be:

Locality-prioritized load balancing

**Comment by howardjohn** [comment]
@geeknoid comments addressed

**Pull request merged**